paymentsheet-example/src/main/res/values/strings.xml

```diff
@@ -16,12 +16,13 @@
     <string name="customer_guest">Guest</string>
     <string name="customer_new">New</string>
     <string name="customer_returning">Returning</string>
+    <string name="link">Link</string>
     <string name="google_pay">Google Pay</string>
     <string name="on">On</string>
     <string name="off">Off</string>
     <string name="checkout_mode">Checkout Mode</string>
     <string name="payment">Pay</string>
-    <string name="payment_with_setup">Pay w/setup</string>
+    <string name="payment_with_setup">P &amp; S</string>
```

Comment:
Can we make this more clear?

Reply:
I'm just trying to save space so the content on the right fits even in smaller screens. I think this is as clear as before given the context.
![image](https://user-images.githubusercontent.com/77990083/162238244-bc8f6dd6-391b-459f-b1b2-273cfb24031c.png)


> Would be nice to make the Pay with Setup more clear. I wonder if it would be better to have Intent: (radio button) Payment (radio button) setup, and if payment is selected a checkbox for On session appears?

Sure, that sounds better, just seems out of the scope of this change.